packages/ses/test/error/test-v8-concise-stack-filtering-unit-tests.js

```diff
@@ -0,0 +1,59 @@
+import test from 'ava';
+import {
+  filterFileName as ff,
+  shortenCallSiteString as scs,
+} from '../../src/error/tame-v8-error-constructor.js';
+
+test('filter file name unit test', t => {
+  t.truthy(ff(undefined));
+  t.falsy(
+    ff(
+      '/Users/markmiller/src/ongithub/agoric/SES-shim/node_modules/ava/lib/runner.js',
+    ),
```

Comment:
It's not clear from the test name which of these should be truthy or falsy.

Ever since using [python's doctest](https://docs.python.org/3/library/doctest.html), I find it frustrating that test frameworks give you a slot for the assertion that you're testing but don't let you put it right next to the docs that say the same thing.

Reply:
Cool! doctest looks like updoc! http://www.erights.org/elang/tools/updoc.html

Added rationale to explanation string of each test case. PTAL.